frontend/.storybook/foundations/Colours/Colours.stories.mdx

```diff
@@ -0,0 +1,240 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks'
```

Comment:
any reason why we're using a mix of story formats?

Reply:
The documentation page looks much better with the MDX format (and much easier for designers to update when needed).

Will probably use MDX for all foundation type components, and CSF for components themselves.